src/mdlComponentHandler.js

```diff
@@ -243,7 +246,7 @@ componentHandler = (function() {
   /**
    * Upgrades a specific list of elements rather than all in the DOM.
    *
-   * @param {!Element|!Array<!Element>|!NodeList|!HTMLCollection} elements
+   * @param {!Element|!Array<!Element>|!NodeList|!HTMLCollection|ShadowRoot} elements
```

Comment:
Can't understand what's wrong with this param and why it isn't compatible with the [line 255](https://github.com/google/material-design-lite/pull/4300/files#diff-8118a6483928b8c7f76329f0d99096b7R255)